packages/cf-style-provider/package.json

```diff
@@ -31,4 +31,4 @@
   "peerDependencies": {
     "react": "^0.14.9 || ^15.0.0-0 || ^16.0.0-0"
   }
-}
+}
```

Comment:
Really feels like it should happen automatically with some thing like [`eol-last`](http://eslint.org/docs/rules/eol-last). No need for a human to add them in manually.